action.yml

```
@@ -351,6 +355,12 @@ runs:
         key: cvmfs-apt-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}-${{ hashFiles('action.yml') }}
         path: |
           ${{ inputs.apt_cache }}
+    - uses: actions/cache@v4
+      if: ${{ inputs.actions_cache == 'true' }}
+      with:
+        key: cvmfs-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}-${{ hashFiles('action.yml') }}
```

```suggestion
        key: cvmfs-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}
```
as seen in #40, using `${{ hashFiles('action.yml') }}` as part of the key is not reliable as the file being hashed as very unlikely to appear in other repositories and causes caches to always miss.